ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -765,6 +765,7 @@ open class DrawerController: UIViewController, TokenizedControlInternal {
             } else {
                 backgroundColor = UIColor(dynamicColor: tokenSet[.drawerContentBackground].dynamicColor)
             }
+            useCustomBackgroundColor = false
```

Comment:
I know why, but it's very odd to be setting `useCustomBackgroundColor = false` in a block that is already `if !useBackgroundColor`. This tells me that there are unintended side effects to setting the background color, which strikes me as fragile. Is there any other way we can factor this code to be less fragile?

If not, I think this at least warrants a comment explaining why it needs to be set here.

Reply:
Talked a bit offline, came up with a good solution. Just removed useCustomBackgroundColor since it's private and moved all the logic to get the different background colors to the getter of backgroundColor.